tests/test_skvbc_linearizability.py

```diff
@@ -21,6 +21,7 @@
 from util import skvbc as kvbc
 from util import skvbc_history_tracker
 from util.bft import with_trio, with_bft_network, KEY_FILE_PREFIX
+from util.skvbc_history_tracker import SkvbcTracker as with_tracker
```

Comment:
this doesn't seem to be used anywhere - also the 'with_tracker' name seems weird...